allan_variance.py

```diff
@@ -67,14 +73,17 @@ def allan_variance(x, dt=1, min_cluster_size=1, min_cluster_count='auto',
     log_max = np.log2((n - min_cluster_count) // 2)
 
     cluster_sizes = np.logspace(log_min, log_max, n_clusters, base=2)
-    cluster_sizes = np.unique(np.round(cluster_sizes)).astype(int)
+    cluster_sizes = np.unique(np.round(cluster_sizes)).astype(np.int64)
```

Comment:
I think the best idea is just not use casting here at all. In the end we use it to scale the floating point numbers.